quilt/tools/build.py

```diff
@@ -37,89 +35,123 @@ def _pythonize_name(name):
         raise BuildException("Unable to determine a Python-legal name for %r" % name)
     return safename
 
-def _build_node(build_dir, package, name, node, fmt, target='pandas'):
+def _run_checks(dataframe, checks, checks_contents, nodename, rel_path, target, env='default'):
+    _ = env  # TODO: env support for checks
+    print("Running data integrity checks...")
+    checks_list = re.split(r'[,\s]+', checks.strip())
+    unknown_checks = set(checks_list) - set(checks_contents.keys())
```

Comment:
I _think_ `.keys()` is not needed

Reply:
NOCHG:

>>> set([3,4,5]) - dict({})
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for -: 'set' and 'dict'

Reply:
I meant, still call `set` - just without the `.keys()`:

```
>>> set([1, 2, 3]) - set({1: 'a', 2: 'b'})
{3}
```

Reply:
DONE.  ah.